.evergreen/scripts/setup_tests.py

```diff
@@ -175,6 +175,28 @@ def handle_test_env() -> None:
         if not config:
             AUTH = "noauth"
 
+    if test_name in ["aws_lambda", "index_management"]:
+        env = os.environ.copy()
+        env["MONGODB_VERSION"] = "7.0"
```

Comment:
Earlier we set the MONGODB_VERSION to 6.0 for `index_management` and this is overriding that from the copy. Do we need to set 6.0 whatsoever?

Reply:
Fixed